documentation/api/admin/v1/delete.markdown

```diff
@@ -0,0 +1,54 @@
+---
+title: "Delete endpoint"
+layout: default
+---
+
+[curl]: ../../query/curl.html#using-curl-from-localhost-non-sslhttp
+
+The `/delete` endpoint can be used to trigger the immediate deletion of all data
+associated with a certname. It is important to note that the delete operation
+doesn't account for in flight commands which may be in the command queue but not
+yet processed by PuppetDB, the targeted node will reappear when the in flight
+command is processed in this case.
+
+## `POST /pdb/admin/v1/delete`
+
+The delete operation must be triggered by a POST.
+
+### Request format
+
+The POST request should specify `Content-Type: application/json` and
+the request body should look like this:
+
+``` json
+{"version" : 1, "command" : "delete", "payload" : {"certname" : CERTNAME_TO_DELETE}}
```

Comment:
```suggestion
{"version" : 1, "command" : "delete", "payload" : {"certname" : "<CERTNAME_TO_DELETE>"}}
```
We use that format elsewhere (ast docs, etc) to signify, it's a string and user supplied.

Reply:
The other style we use in command documentation is
```suggestion
{"version" : 1, "command" : "delete", "payload" : {"certname" : <string>}}
```

Reply:
👍 I'm cool with the second one